.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
   check-code-formatting:
     name: Check code formatting
     runs-on: ubuntu-latest
-    container: openrct2/openrct2-build:4-format
+    container: openrct2/openrct2-build:17-format
```

Comment:
This refers to v17, but the check-code-formatting script refers to v18. Is that intended?

Reply:
17 is just the version of our docker container, as far as I'm aware.